cluegonauts/clueless/templates/clueless/gameb.html

```diff
@@ -21,21 +21,36 @@
                 {% for row in locations %}
                     {% for location in row %}
                         {% if location.location_id in blank_ids %}
-                            <div id="{{ location.location_id }}_card" class="card py-20 rounded-lg opacity-50 w-auto">
+                            <div id="{{ location.location_id }}_card" class="card opacity-0 w-auto">
                                 <div class="card-body" id="{{ location.location_id }}">
                                     <h2 class="card-title">{{ location.name }}</h2>
                                 </div>
                             </div>
-                        {% elif location.location_id in hallway_ids  %}
-                            <div id="{{ location.location_id }}_card" class="card py-20 rounded-lg bg-gray-500 opacity-55 w-auto inline items-center" role="button">
+                        {% elif location.location_id in hallway_ids %}
+                            <div id="{{ location.location_id }}_card" class="card image-full w-auto inline items-center" role="button">
```

Comment:
@dargos82 I think the gameboard looks fantastic! But I would recommend adding the opacity back in - the icons are impossible to see without that. But other than that I think this looks great!
One note which maybe @bruszow1 has some related changes for - the clicking in the right spot to move issue seems to be worse, probably because of the addition of the image. Maybe the image element needs the id that is recognized on click? May be solved by Boleslaw's changes though.

Reply:
@arorakarnika I thought the same about the clicking - it seemed more difficult to find the "correct" spot.

Opacity adjusted and this branch is rebased so conflicts should be resolved.

![Screenshot 2024-12-07 at 11 33 18 PM](https://github.com/user-attachments/assets/9026fe5d-fcfe-4bfd-840d-62a44b1bfbb1)

Reply:
The image element blocks the card div element so my changes won't fix it. Since the image completely covers the card div element, the id currently used for the card div can be moved to the image element.

If going that route, the line in the JavaScript function linked below needs to be changed.
https://github.com/arorakarnika/Cluegonauts/blob/cab0e4bf0f1e73d492935fbde45babe145343034/cluegonauts/clueless/templates/clueless/gameb.html#L238
Instead of looking for a child element, you can remove the "_card" suffix from the id with `location = e.target.id.replace('_card', '');`

Reply:
@bruszow1 and @arorakarnika Thanks, I have a couple questions from your feedback.  I interpret it to mean two things: 1) remove the "_card" suffix at line 237 and 2) update the location id at line 238.  So it would look like this?

 236       // if clicked on card, get child element
 237       if (e.target.id.includes) {
 238           location = e.target.id.replace('_card', ' ');